src/openrct2/peep/peep.c

```diff
@@ -12932,3 +12917,17 @@ void decrement_guests_heading_for_park(){
         log_error("Attempt to decrement guests heading for park below zero.");
     }
 }
+
+void release_balloon(rct_peep *peep) {
```

Comment:
peep_release_balloon would be more consistent with other function names.

Also, if this function is not used outside peep.c, you can declare it static.